gym/envs/tests/test_envs.py

```diff
@@ -12,7 +12,12 @@ def should_skip_env_spec_for_tests(spec):
     # We skip tests for envs that require dependencies or are otherwise
     # troublesome to run frequently
 
-    # Skip mujoco tests for pull request CI
+    # Skipping if pull request
+    is_pr = os.environ.get('TRAVIS_PULL_REQUEST') != 'false' and os.environ.get('TRAVIS_PULL_REQUEST') is not None
```

Comment:
Ah ha. I just changed the .travis.yml to only create it if `MUJOCO_KEY_BUNDLE` is set (which is true for non-PR builds).